reactjs-films-homework/src/languages/getLanguage.ts

```diff
@@ -0,0 +1,26 @@
+import data from './languagesData';
+
+const lang = (language: string) => {
```

Comment:
the name of the function should start from the verb. For example: 'getTranslation'.